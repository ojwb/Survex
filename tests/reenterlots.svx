; pos=no warn=6 error=0
*fix svy.1 0 0 0
*begin svy
*export 1
1 2 10.00 000 0
*end svy
*begin svy
1 3 10.00 060 0
*end svy
*begin svy
1 4 10.00 120 0
*end svy
*begin svy
1 5 10.00 180 0
*end svy
*begin svy
1 6 10.00 240 0
*end svy
*begin svy
1 7 10.00 300 0
*end svy
*begin svy
1 8 10.00 - down
*end svy
*begin svy
1 9 10.00 - up
*end svy
