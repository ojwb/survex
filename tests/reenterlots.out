./reenterlots.svx:7:8: warning: Reentering an existing survey is deprecated
 *begin svy
        ^~~
./reenterlots.svx:3:8: info: Originally entered here
./reenterlots.svx:10:8: warning: Reentering an existing survey is deprecated
 *begin svy
        ^~~
./reenterlots.svx:3:8: info: Originally entered here
./reenterlots.svx:13:8: warning: Reentering an existing survey is deprecated
 *begin svy
        ^~~
./reenterlots.svx:3:8: info: Originally entered here
./reenterlots.svx:16:8: warning: Reentering an existing survey is deprecated
 *begin svy
        ^~~
./reenterlots.svx:3:8: info: Originally entered here
./reenterlots.svx:19:8: warning: Reentering an existing survey is deprecated
 *begin svy
        ^~~
./reenterlots.svx:3:8: info: Originally entered here
./reenterlots.svx:19: info: Further uses of this deprecated feature will not be reported
 *begin svy

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 9 survey stations, joined by 8 legs.
There are 0 loops.
Total length of survey legs =   80.00m (  80.00m adjusted)
Total plan length of survey legs =   60.00m
Total vertical length of survey legs =   20.00m
Vertical range = 20.00m (from svy.9 at 10.00m to svy.8 at -10.00m)
North-South range = 20.00m (from svy.2 at 10.00m to svy.5 at -10.00m)
East-West range = 17.32m (from svy.4 at 8.66m to svy.7 at -8.66m)

There were 5 warning(s).
